scripts/scil_volume_resample.py

```diff
@@ -84,9 +84,20 @@ def main():
 
     img = nib.load(args.in_image)
 
+    ref_img = None
+    if args.ref:
+        ref_img = nib.load(args.ref)
+        # Must not verify that headers are compatible. But can verify that, at
+        # least, that the last column of their affine are compatible.
```

Comment:
Remove this `that`. It is already in the previous line.

Reply:
Also, `is compatible`

Reply:
I think it's rather `affines are compatible`